tekton_configs/test-run.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-testing
+  namespace: gabrielwen
+spec:
+  pipelineSpec:
+   tasks:
+   - name: xgboost-synthetic
+     taskRef:
+      name: pytest
+     params:
+     - name: src-dir
+       value: /mnt/testdir/src/gabrielwen
+     - name: repos
+       value: kubeflow/examples@HEAD,kubeflow/testing@HEAD,kubeflow/tf-operator@HEAD
+     - name: python-path
+       value: /mnt/testdir/src/gabrielwen/kubeflow/examples/py:/mnt/testdir/src/gabrielwen/kubeflow/testing/py:/mnt/testdir/src/gabrielwen/kubeflow/tf-operator/py
+     - name: get-cred-args-array
+       value:
+       - "-m"
+       - "kubeflow.testing.get_kf_testing_cluster"
+       - "get-credentials"
+     - name: pytest-args-array
+       value:
+        - xgboost_test.py
```

Comment:
Can we reduce the number of parameters that need to be supplied for the task?

Reply:
yes we definitely should.  this is a draft so just wanna give people a taste of this.